### PR TITLE
(DOCUMENT-458) Fix site.pp paths

### PR DESCRIPTION
**Pull request by gguillotte**
Paths to `site.pp` files are incorrect in examples across versions of the Puppet docs. Fix these paths to the appropriate locations for each version.

### CODE REVIEW

File: source/puppet/4.1/reference/dirs_manifest.markdown [hunk 1]
```diff
@@ -17,23 +17,21 @@ canonical: "/puppet/latest/reference/dirs_manifest.html"
...
+For more information on how the site manifest is used in catalog compilation, see [the reference page on catalog compilation][catalog_compilation].
 
-Location
------
+## Location
 
 ### With Puppet Apply
 
-The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/puppet/manifests/site.pp`.) It may be a single file or a directory of files.
+The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/code/environments/production/site.pp`.) It can be a single file or a directory of files.
```
Comment (nfagerlund) [inline source/puppet/4.1/reference/dirs_manifest.markdown]: /etc/puppetlabs/code/environments/production/manifests/site.pp

File: source/puppet/4.1/reference/modules_fundamentals.markdown [hunk 2]
```diff
@@ -37,27 +34,26 @@ Every Puppet user should expect to write at least some of their own modules.
...
 
-Using Modules
------
+## Using Modules
 
 Modules are how Puppet finds the classes and types it can use --- it automatically loads any [class][classes] or [defined type][defined_types] stored in its modules. Any of these classes or defines can be declared by name within a manifest or from an [external node classifier (ENC)][enc].
 
 ~~~ ruby
-    # /etc/puppetlabs/puppet/site.pp
+# /etc/puppetlabs/code/environments/production/site.pp
```
Comment (nfagerlund) [inline source/puppet/4.1/reference/modules_fundamentals.markdown]: again, needs a .../manifests/... subdirectory.

File: source/puppet/4.2/reference/dirs_manifest.markdown [hunk 3]
```diff
@@ -17,23 +17,21 @@ canonical: "/puppet/latest/reference/dirs_manifest.html"
...
+For more information on how the site manifest is used in catalog compilation, see [the reference page on catalog compilation][catalog_compilation].
 
-Location
------
+## Location
 
 ### With Puppet Apply
 
-The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/puppet/manifests/site.pp`.) It may be a single file or a directory of files.
+The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/code/environments/production/site.pp`.) It can be a single file or a directory of files.
```
Comment (nfagerlund) [inline source/puppet/4.2/reference/dirs_manifest.markdown]: needs .../manifests/...

File: source/puppet/4.2/reference/modules_fundamentals.markdown [hunk 4]
```diff
@@ -37,27 +34,26 @@ Every Puppet user should expect to write at least some of their own modules.
...
 
-Using Modules
------
+## Using Modules
 
 Modules are how Puppet finds the classes and types it can use --- it automatically loads any [class][classes] or [defined type][defined_types] stored in its modules. Any of these classes or defines can be declared by name within a manifest or from an [external node classifier (ENC)][enc].
 
 ~~~ ruby
-    # /etc/puppetlabs/puppet/site.pp
+# /etc/puppetlabs/code/environments/production/site.pp
```
Comment (nfagerlund) [inline source/puppet/4.2/reference/modules_fundamentals.markdown]: needs .../manifests/...

File: source/puppet/4.3/reference/dirs_manifest.markdown [hunk 5]
```diff
@@ -17,23 +17,21 @@ canonical: "/puppet/latest/reference/dirs_manifest.html"
...
+For more information on how the site manifest is used in catalog compilation, see [the reference page on catalog compilation][catalog_compilation].
 
-Location
------
+## Location
 
 ### With Puppet Apply
 
-The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/puppet/manifests/site.pp`.) It may be a single file or a directory of files.
+The `puppet apply` command requires a manifest as an argument on the command line. (For example: `puppet apply /etc/puppetlabs/code/environments/production/site.pp`.) It can be a single file or a directory of files.
```
Comment (nfagerlund) [inline source/puppet/4.3/reference/dirs_manifest.markdown]: needs .../manifests/...

File: source/puppet/4.3/reference/modules_fundamentals.markdown [hunk 6]
```diff
@@ -37,27 +34,26 @@ Every Puppet user should expect to write at least some of their own modules.
...
 
-Using Modules
------
+## Using Modules
 
 Modules are how Puppet finds the classes and types it can use --- it automatically loads any [class][classes] or [defined type][defined_types] stored in its modules. Any of these classes or defines can be declared by name within a manifest or from an [external node classifier (ENC)][enc].
 
 ~~~ ruby
-    # /etc/puppetlabs/puppet/site.pp
+# /etc/puppetlabs/code/environments/production/site.pp
```
Comment (nfagerlund) [inline source/puppet/4.3/reference/modules_fundamentals.markdown]: needs .../manifests/...

**Comment by nfagerlund** [comment]
One new typo in several places, but that's the only issue I could see.

**Comment by gguillotte** [comment]
@nfagerlund Should be addressed by the new commit.

**Pull request merged**